lib/IlmCtl/CtlInterpreter.cpp

```diff
@@ -75,9 +76,11 @@
 #include <memory>
 
 #ifdef WIN32
-    #include <io.h>
     #include <stdio.h>
     #include <stdlib.h>
+    #include <io.h>
+    #include <time.h>
+    #define lrand48  rand
```

Comment:
@AntonQubecinema I'm concerned about the upper limit `rand()` will be able to provide. I will be better to provide windows-specific compatible implementation of `lrand48()`